build_tools/github/check_wheels.py

```diff
@@ -20,15 +20,6 @@
 # plus one more for the sdist
 n_wheels += 1
 
-# aarch64 builds from travis
-travis_config_path = Path.cwd() / ".travis.yml"
-with travis_config_path.open("r") as f:
-    travis_config = yaml.safe_load(f)
-
-jobs = travis_config["jobs"]["include"]
-travis_builds = [j for j in jobs if any("CIBW_BUILD" in env for env in j["env"])]
-n_wheels += len(travis_builds)
-
```

Comment:
I think that we have to include here the number of wheels for circleci.

Or should this be done in a different PR?

Reply:
That was my first guess then I read [this comment](https://github.com/scikit-learn/scikit-learn/pull/19060#issuecomment-749600129) and I was lost... @ogrisel, please let us know, what should be done here. Thanks!

Reply:
We do need to count the 3 linux arm64 wheels in the `build_tools/github/check_wheels.py ` script prior to uploading everything to pypi.

Reply:
The point of [my comment](https://github.com/scikit-learn/scikit-learn/pull/19060#issuecomment-749600129) was to avoid counting the linux arm64 wheels twice in 2 almost concurrently merged PRs.

Reply:
Thanks, this wasn't clear to me.

Reply:
Thanks @alfaro96 for bringing this up. I have added the check. I'm resolving this conversation. Feel free to comment in the new version of the file.